src/libstd/lib.rs

```diff
@@ -419,6 +420,11 @@ pub use core::raw;
 pub use core::result;
 #[stable(feature = "rust1", since = "1.0.0")]
 pub use core::option;
+#[unstable(feature = "c_variadic",
+           reason = "the `c_variadic` feature has not been properly tested on \
+                     all supported platforms",
+           issue = "27745")]
+pub use core::va_list;
```

Comment:
I wonder if there is a better place to put this in.

Reply:
Yeah one of my original comments on this PR was about that. The original RFC indicates that the structure would be in `core::intrinsics`, but that also seemed strange to me. Any feedback on this would be appreciated. Maybe something like `ffi`, `cffi`, or `cabi`?

Reply:
I'd say `ffi` assuming `std::ffi` would export it (and we don't involve `libc`).